### ISSUE TITLE
Do live dials cause windows to pop up?

### ISSUE THREAD

**Issue by w-u-2-o** [fastaddons/GroupSpeedDial#200]
Using Brave browser. When I refresh a dial a window pops up that loads the page. Does this also happen with live dials?

**Comment by fastaddons** [comment]
Refreshing a dial needs to load the page in order to create a thumbnail.
Sadly, in Chromium-based browsers like Brave or Chrome or Edge the reloading window can't be hidden anymore. That's why it pops up. Only Firefox supports invisible thumbnails generation, for now.

A "Live dial" loads a whole page into the dial so it's not exactly a replacement. Loading a full page into small dial takes a lot of resources. So if you have 5 live dials in one group, it's like opening 5 pages at the same time.

**Comment by w-u-2-o** [comment]
Looking at the animated .gif you referenced, it would appear that Live Dials only go live when you mouse over them. Is that correct?

What I'm looking for is an occasional, scheduled update of the dial so I know whether or not anything has changed on the page that would make me want to go open it.

**Comment by fastaddons** [comment]
Nope, the page is fully live. It's being load inside a special element called "iframe" which simply loads a whole page into this container.

But not every page can be loaded into iframe, but those are very rare.
You can test it by opening "Edit dial" window / "Crop scale" / Load preview button. If the page can load there, it can load as live dial.

**Comment by w-u-2-o** [comment]
Thank you for the explanation, Juraj.
